_dev/src/store/modules/product-feed/actions.ts

```diff
@@ -242,12 +252,12 @@ export default {
     const deliveryFromStorage = getDataFromLocalStorage('productFeed-deliveryDetails') ?? [];
 
     if (state.settings.shippingSetup === ShippingSetupOption.ESTIMATE) {
-      const getEstimateCarrier = getDataFromLocalStorage('productFeed-customCarrier');
+      const getEstimateCarrier = getDataFromLocalStorage('productFeed-estimateCarriers');
 
       if (getEstimateCarrier !== null) {
         commit(MutationsTypes.SET_SELECTED_PRODUCT_FEED_SETTINGS, {
           name: 'estimateCarrier',
-          data: getEstimateCarrier,
+          data: getEstimateCarrier[0],
```

Comment:
If you want your code to be clean and stable, future proof with the multi target countries, consider having an array in your state as well.